### ISSUE TITLE
Empty screen

### ISSUE THREAD

**Issue by crossrw** [DonJayamanne/gitHistoryVSCode#552]
After updating, the screen is empty when executing any of the commands "Git: View History", "Git: View File History". Reinstalling and going back to an older version doesn't help.

**Comment by ole1986** [comment]
@crossrw you may also open the Webview developer tools and check for error.
Also please tell us more about OS, vscode version, etc..

Since I am using latest vscode with current git history version I also noticed some strange behavior on mac os.

Possible related to:
https://github.com/microsoft/vscode/issues/98231
https://github.com/microsoft/vscode/issues/89038
https://github.com/microsoft/vscode/pull/97777

**Comment by crossrw** [comment]
Hi @ole1986, thanks for the tip, versions:

vscode version 1.45.1
node.js 12.8.1
OS: Windows_NT x64 6.1.7601
Git History: 0.6.5

Webview developer tools console output:

```
data:text/html;char…:3 Uncaught SyntaxError: Invalid Unicode escape sequence
bundle.js:31 TypeError: Cannot read property 'pageSize' of undefined
    at he.render (bundle.js:47)
    at Vi (bundle.js:31)
    at Ai (bundle.js:31)
    at ml (bundle.js:31)
    at us (bundle.js:31)
    at ls (bundle.js:31)
    at Jl (bundle.js:31)
    at ql (bundle.js:31)
    at Gs (bundle.js:31)
    at bundle.js:31
el @ bundle.js:31
bundle.js:31 Uncaught TypeError: Cannot read property 'pageSize' of undefined
    at he.render (bundle.js:47)
    at Vi (bundle.js:31)
    at Ai (bundle.js:31)
    at ml (bundle.js:31)
    at us (bundle.js:31)
    at ls (bundle.js:31)
    at Jl (bundle.js:31)
    at ql (bundle.js:31)
    at Gs (bundle.js:31)
    at bundle.js:31
```

**Comment by ole1986** [comment]
@crossrw any chance to get the repository to test it on our side?

**Comment by fangguangyang** [comment]
@ole1986 I also encountered this problem
 
version: 1.46.0-insider (user setup)
commit: 2591ede95a52400d5d22989b16706f9b8b4831da
date: 2020-05-22T21:13:15.709Z
Electron: 8.3.0
Chrome: 80.0.3987.165
Node.js: 12.13.0
V8: 8.0.426.27-electron.0
OS: Windows_NT x64 10.0.18363

Git History: 0.6.5

It's normal on local windows system, but I use it on remote-ssh+centos7 is abnormal

![image](https://user-images.githubusercontent.com/14014903/82769095-86486b80-9e65-11ea-8931-e09eaa1673a2.png)
![image](https://user-images.githubusercontent.com/14014903/82769098-8d6f7980-9e65-11ea-9f42-07881fbaf460.png)

**Comment by ole1986** [comment]
@fangguangyang can you please provide us the console output of it **webview** developer tool as described above?

**Comment by fangguangyang** [comment]
@ole1986 

Hi, Do you mean these?

console.ts:137 [Extension Host] (node:14996) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead. (at writeOut (internal/process/warning.js:27:3))
t.log @ console.ts:137
console.ts:137 [Extension Host] (node:14996) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead. (at writeOut (internal/process/warning.js:27:3))
t.log @ console.ts:137
TMScopeRegistry.ts:45 Overwriting grammar scope name to file mapping for scope source.nginx.
Old grammar file: file:///c%3A/Users/BrainiacLegion/.vscode-insiders/extensions/william-voyek.vscode-nginx-0.7.2/syntaxes/nginx.tmLanguage.json.
New grammar file: vscode-remote://ssh-remote%2B7b22686f73744e616d65223a2269436f7265227d/root/.vscode-server-insiders/extensions/hangxingliu.vscode-nginx-conf-hint-0.1.0/syntaxes/nginx.tmLanguage
register @ TMScopeRegistry.ts:45
TMScopeRegistry.ts:45 Overwriting grammar scope name to file mapping for scope source.lua.
Old grammar file: file:///c%3A/Users/BrainiacLegion/AppData/Local/Programs/Microsoft%20VS%20Code%20Insiders/resources/app/extensions/lua/syntaxes/lua.tmLanguage.json.
New grammar file: vscode-remote://ssh-remote%2B7b22686f73744e616d65223a2269436f7265227d/root/.vscode-server-insiders/extensions/sumneko.lua-0.17.1/syntaxes/lua.tmLanguage.json
register @ TMScopeRegistry.ts:45
searchService.ts:217 No search provider registered for scheme: vscode-remote, waiting
(anonymous) @ searchService.ts:217
DevTools failed to parse SourceMap: file:///C:/Users/BrainiacLegion/AppData/Local/Programs/Microsoft%20VS%20Code%20Insiders/resources/app/out/vs/code/electron-browser/workbench/index.js.map
DevTools failed to parse SourceMap: file:///C:/Users/BrainiacLegion/AppData/Local/Programs/Microsoft%20VS%20Code%20Insiders/resources/app/out/vs/code/electron-browser/workbench/windowsTerminal.js.map
DevTools failed to parse SourceMap: file:///C:/Users/BrainiacLegion/AppData/Local/Programs/Microsoft%20VS%20Code%20Insiders/resources/app/out/vs/code/electron-browser/workbench/terminal.js.map
DevTools failed to parse SourceMap: file:///C:/Users/BrainiacLegion/AppData/Local/Programs/Microsoft%20VS%20Code%20Insiders/resources/app/out/vs/code/electron-browser/workbench/eventEmitter2.js.map
DevTools failed to parse SourceMap: file:///C:/Users/BrainiacLegion/AppData/Local/Programs/Microsoft%20VS%20Code%20Insiders/resources/app/out/vs/code/electron-browser/workbench/windowsPtyAgent.js.map
DevTools failed to parse SourceMap: file:///C:/Users/BrainiacLegion/AppData/Local/Programs/Microsoft%20VS%20Code%20Insiders/resources/app/out/vs/code/electron-browser/workbench/utils.js.map
DevTools failed to parse SourceMap: file:///C:/Users/BrainiacLegion/AppData/Local/Programs/Microsoft%20VS%20Code%20Insiders/resources/app/out/vs/code/electron-browser/workbench/index.js.map
DevTools failed to parse SourceMap: file:///C:/Users/BrainiacLegion/AppData/Local/Programs/Microsoft%20VS%20Code%20Insiders/resources/app/out/vs/code/electron-browser/workbench/xterm.js.map
DevTools failed to parse SourceMap: file:///C:/Users/BrainiacLegion/AppData/Local/Programs/Microsoft%20VS%20Code%20Insiders/resources/app/out/vs/code/electron-browser/workbench/xterm-addon-unicode11.js.map
DevTools failed to parse SourceMap: file:///C:/Users/BrainiacLegion/AppData/Local/Programs/Microsoft%20VS%20Code%20Insiders/resources/app/out/vs/code/electron-browser/workbench/xterm-addon-search.js.map
DevTools failed to parse SourceMap: file:///C:/Users/BrainiacLegion/AppData/Local/Programs/Microsoft%20VS%20Code%20Insiders/resources/app/out/vs/code/electron-browser/workbench/index.js.map

**Comment by ole1986** [comment]
Yes, but please first run the command that causes the error and then output the error..

**Comment by fangguangyang** [comment]
I only get this when I run the  command `Git: View File History`

```
deprecate.ts:33 (electron) 'webview.getWebContents()' is deprecated and will be removed. Please use 'remote.webContents.fromId(webview.getWebContentsId())' instead.
log @ deprecate.ts:33
(anonymous) @ deprecate.ts:11
(anonymous) @ deprecate.ts:61
get webContents @ webviewElement.ts:65
(anonymous) @ webviewElement.ts:45
(anonymous) @ functional.ts:17
dispatchEvent @ web-view-impl.ts:125
dispatchEvent @ guest-view-internal.ts:59
(anonymous) @ guest-view-internal.ts:77
emit @ events.js:210
onMessage @ init.ts:59
```

**Comment by ole1986** [comment]
Thats odd! Since webview.getWebContents() is not used by Git History as far as I can see.
Have you tried disabling all other extensions except "Git History" and tried it again?

**Comment by fangguangyang** [comment]
@ole1986 Hi, I don't think this is the reason.
this extensions worked very well  when I didn't update vscode-insider.
but I use the last version of vscode-insider, it broke this extensions.

so, absolutely the reason is new version of vscode-insider.

**Comment by qadmus** [comment]
I'm having the same behavior with all other extensions disabled, on Windows:

```
[Embedded Page] Uncaught SyntaxError: Invalid Unicode escape sequence | webviewElement.ts:331
[Embedded Page] TypeError: Cannot read property 'pageSize' of undefined | webviewElement.ts:331
[Embedded Page] Uncaught TypeError: Cannot read property 'pageSize' of undefined | webviewElement.ts:331
```

**Comment by cameleon77** [comment]
Hi, the same empty page here, not able to see the log (Git View File History).

```
Version: 1.46.1 (user setup)
Commit: cd9ea6488829f560dc949a8b2fb789f3cdc05f5d
Date: 2020-06-17T21:13:20.174Z
Electron: 7.3.1
Chrome: 78.0.3904.130
Node.js: 12.8.1
V8: 7.8.279.23-electron.0
OS: Windows_NT x64 10.0.18363
```


```
Uncaught SyntaxError: Invalid Unicode escape sequence
bundle.js:31 TypeError: Cannot read property 'pageSize' of undefined
    at he.render (bundle.js:47)
    at Vi (bundle.js:31)
    at Ai (bundle.js:31)
    at ml (bundle.js:31)
    at us (bundle.js:31)
    at ls (bundle.js:31)
    at Jl (bundle.js:31)
    at ql (bundle.js:31)
    at Gs (bundle.js:31)
    at bundle.js:31
el @ bundle.js:31
bundle.js:31 Uncaught TypeError: Cannot read property 'pageSize' of undefined
    at he.render (bundle.js:47)
    at Vi (bundle.js:31)
    at Ai (bundle.js:31)
    at ml (bundle.js:31)
    at us (bundle.js:31)
    at ls (bundle.js:31)
    at Jl (bundle.js:31)
    at ql (bundle.js:31)
    at Gs (bundle.js:31)
    at bundle.js:31
```

**Comment by ole1986** [comment]
This is possibly fixed with PR #563 
Feel free to reopen if its not resolve with the upcoming version (0.6.7)
